azurerm/internal/services/logic/logic_app_action_http_resource.go

```diff
@@ -99,6 +123,10 @@ func resourceArmLogicAppActionHTTPCreateUpdate(d *schema.ResourceData, meta inte
 		"type":   "http",
 	}
 
+	if v, ok := d.GetOk("run_after"); ok {
+		action["runAfter"] = expandLogicAppActionRunAfter(v.(*schema.Set).List())
+	}
```

Comment:
do we need to set this to an empty array if this is unset?

Reply:
The test result shows that is not necessary. Also in the previous case where we didn't set it at all, it still works for the first level action (which is right after the trigger).